### PR TITLE
paginatedmessages: switch to MessageSend

### PR DESCRIPTION
**Pull request by SoggySaussages**
Previous paginated messages were limited to message embeds only. This pr switches them over to MessageSend struct, allowing full customization.

If the paginated message contains embeds, the page number and timecode are added to the final embed. If the message contains no embeds, the page number is appended to the message content.

The paginate test command is added to demonstrate this functionality, with an optional arg to control page count, and a switch for demonstrating messages without embeds.

Previous paginated commands are also updated to use MessageSend instead.

Screenshot 2022-11-28 at 01 18 59

### CODE REVIEW

**Comment by lemm-e** [comment]
I don't quite understand the point of PR'ing a test command to the official repository.

File: stdcommands/paginatetest/paginatetest.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package paginatetest
```
Comment (ashishjh-bst) [inline stdcommands/paginatetest/paginatetest.go:1]: @SoggySaussages this isn't needed.

File: stdcommands/stdcommands.go [hunk 2]
```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/botlabs-gg/yagpdb/v2/stdcommands/memstats"
 	"github.com/botlabs-gg/yagpdb/v2/stdcommands/owldictionary"
 	"github.com/botlabs-gg/yagpdb/v2/stdcommands/ping"
+	"github.com/botlabs-gg/yagpdb/v2/stdcommands/paginatetest"
```
Comment (ashishjh-bst) [inline stdcommands/stdcommands.go:36]: remove this to along with the paginatetest command.

Comment (SoggySaussages) [reply stdcommands/stdcommands.go]: Sounds good, thanks

**Comment by mrbentarikau** [comment]
I still do not see any use cases for plain text page numbers here, only bloating already existing code with additional code, examples like `return &discordgo.MessageSend{Embeds: []*discordgo.MessageEmbed{paginatedEmbed}}, nil` instead of `return paginatedEmbed. nil`

**Comment by ashishjh-bst** [comment]
The plain text pagination can be handy in commands like cc, topcommands, topservers. 
I'll try out some experiments with it before v2.14.0

**Comment by mrbentarikau** [comment]
but why change all embed responses thing for this, add new pagination funcs for content messages... these are not even that necessary, you can embed-paginate all the commands you listed

**Comment by ashishjh-bst** [comment]
I'll be closing this because there isn't really a need to move to sendmessages as mrben has said, and the original issue for which this was created has been fixed in a different PR.

**Pull request closed without merging**